src/nvim/main.c

```diff
@@ -119,25 +119,21 @@ typedef struct {
 # include "main.c.generated.h"
 #endif
 
-/*
- * Different types of error messages.
- */
-static char *(main_errors[]) =
-{
+// Error messages
+static const char *main_errors[] = {
   N_("Unknown option argument"),
-#define ME_UNKNOWN_OPTION       0
   N_("Too many edit arguments"),
-#define ME_TOO_MANY_ARGS        1
   N_("Argument missing after"),
-#define ME_ARG_MISSING          2
   N_("Garbage after option argument"),
-#define ME_GARBAGE              3
-  N_("Too many \"+command\", \"-c command\" or \"--cmd command\" arguments"),
-#define ME_EXTRA_CMD            4
-  N_("Invalid argument for"),
-#define ME_INVALID_ARG          5
+  N_("Too many \"+command\", \"-c command\" or \"--cmd command\" arguments")
 };
 
+#define ME_UNKNOWN_OPTION  0
+#define ME_TOO_MANY_ARGS   1
+#define ME_ARG_MISSING     2
+#define ME_GARBAGE         3
+#define ME_EXTRA_CMD       4
```

Comment:
I think it was useful when these defines lived next to their respective messages.